src/index.html

```diff
@@ -9,10 +9,44 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
Link to a SCSS file should be replaced with a compiled CSS file. Browsers do not understand SCSS syntax, they require plain CSS. Change the href attribute to point to the corresponding CSS file.